src/librustdoc/html/render.rs

```diff
@@ -1460,7 +1460,8 @@ impl<'a> fmt::Display for Item<'a> {
         try!(write!(fmt, "<span class='out-of-band'>"));
         try!(write!(fmt,
         r##"<span id='render-detail'>
-            <a id="toggle-all-docs" href="#" title="collapse all docs">[&minus;]</a>
+            <a id="toggle-all-docs" href="javascript:void(0)"
+               title="collapse all docs">[&#x2212;]</a>
```

Comment:
Hmm I suppose this interferes with having a single uniform declaration of this :/